examples/src/main/java/org/apache/ignite/example/table/KeyValueBinaryViewExample.java

```diff
@@ -43,72 +46,71 @@
  */
 public class KeyValueBinaryViewExample {
     public static void main(String[] args) throws Exception {
-        Ignite ignite = IgnitionManager.start(
+        try (Ignite ignite = IgnitionManager.start(
             "node-0",
-            Files.readString(Path.of("config", "ignite-config.json").toAbsolutePath()),
+            Files.readString(Path.of("config", "ignite-config.json")),
```

Comment:
In `SqlJdbcExample` we are using path `"examples/config/ignite-config.json"`. Need to unify it (change path here or there)

Reply:
Made it the same in SqlJdbcExample.